Makefile

```diff
@@ -64,3 +68,13 @@ build:
 		--args.input.dir="$(REPODIR)" \
 		--propertyfile="$(DITA_PROPERTY_FILE)"
 	cp -r $(REPODIR)/resources/ $(GT_DOC_OUT)
+
+# Consistently indent XML
+format-xml:
+	find . -path ./$(notdir $(DITA_OT_DIR)) -prune -false -o -name '*.dita' -a -not \( -size 0 \) |while read i;do \
+		echo xmlstarlet fo $$i > $$i.formatted && mv $$i.formatted $$i; \
```

Comment:
Forgive my being inquisitive, but what's `-false -o` for? IIUC, `-prune` always evaluates to true, then `-false` always to false, then `-o` is a disjunction, then comes `-name`. So IINM that together means the same as `-prune -name '*.dita'`.

For `\( -size 0 \)` there's also `-empty` BTW.

The shell's while loop could be rewritten as ` | xargs 'xmlstarlet fo {} | sponge {}'` or ` | parallel -j 'xmlstarlet fo {} | sponge {}'` BTW.

Reply:
Inquisitiveness is appreciated here.

`-prune -false -o` is one of those snippets I have in muscle memory. It doesn't do the right thing without `-false -o`. I agree it should but it doesn't. `find(1)` is weird sometimes.

`\( -size 0 \)` == `-empty` thanks, I'll change it.

As for `sponge`/`parallel`: Thanks for the tip, `sponge` solves the redirection issue and your approach is probably more efficient, but the `while` loop is easier to debug. And as for "debugging" I just noticed I committed the version with `echo`. My bad, will fix.